packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1711,6 +1711,7 @@ export type ChatCompositeOptions = {
     topic?: boolean;
     autoFocus?: 'sendBoxTextField';
     fileSharing?: FileSharingOptions;
+    richTextEditor?: boolean | RichTextEditorOptions;
```

Comment:
Do we have other previous use case similar to this boolean | RichTextEditorOptions type?
Why don't we just use RichTextEditorOptions here and since this prop is optional, if it's undefined, we don't enable richTextEditor?
This is what we do for Mention currently.

Reply:
That's a good question. I was following what we have done for some features on the Call Composite

<img width="844" alt="image" src="https://github.com/Azure/communication-ui-library/assets/73612854/723cb964-8164-44aa-82fd-0570e1ef4f87">

<img width="656" alt="image" src="https://github.com/Azure/communication-ui-library/assets/73612854/3ab3f970-dc9e-436c-9d57-72a282652da0">

Although it's not very consistent. As also have these apis that have different format for doing the same thing.

<img width="509" alt="image" src="https://github.com/Azure/communication-ui-library/assets/73612854/e76ac81c-37a9-490c-926e-2c08a7d668c5">

<img width="429" alt="image" src="https://github.com/Azure/communication-ui-library/assets/73612854/1c081fc9-cf00-49d8-bfbd-673751c3a40c">

At least for now this was mainly mean as stub API for testing purposes that we would later build on.